components/net/bluetooth_thread.rs

```diff
@@ -452,6 +458,11 @@ impl BluetoothManager {
             Some(a) => a,
             None => return drop(sender.send(Err(String::from(ADAPTER_ERROR)))),
         };
+        if let Some(ref id) = uuid {
```

Comment:
My other concern is that we `match` the `uuid` 2 times instead of 1.  see `line 466`